pkg/hintrunner/zero/zerohint.go

```diff
@@ -73,6 +84,95 @@ func CreateAllocSegmentHinter(resolver hintReferenceResolver) (hinter.Hinter, er
 	return &core.AllocSegment{Dst: hinter.ApCellRef(0)}, nil
 }
 
+func createIsLeFeltHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
```

Comment:
This file is going to get big too fast. Other than what we had discussed to do Unit testing, we could also consider having separated files to group set of hints (as they are separated in the issue #164 for example). In this case, they could go in a `math.go` file.

Reply:
I'll move it in the upcoming PR.